i18n/src/main/res/values-zh-rCN/feat_stream.xml

```diff
@@ -4,18 +4,22 @@
     <string name="feat_channel_playback_state_buffering">缓冲中</string>
     <string name="feat_channel_playback_state_ready">就绪</string>
     <string name="feat_channel_playback_state_ended">结束</string>
+    
     <string name="feat_channel_tooltip_on_back_pressed">返回</string>
     <string name="feat_channel_tooltip_mute">静音</string>
     <string name="feat_channel_tooltip_unmute">取消静音</string>
     <string name="feat_channel_tooltip_favourite">收藏</string>
     <string name="feat_channel_tooltip_unfavourite">取消收藏</string>
     <string name="feat_channel_tooltip_download">下载</string>
     <string name="feat_channel_tooltip_stop_download">停止下载</string>
-    <string name="feat_channel_tooltip_cast">投屏</string>
+    <string name="feat_channel_tooltip_cast">投屏 Cast</string>
```

Comment:
🤔

Reply:
You'd better revert all mainland translation if you do not learn more about it.

Reply:
@oxyroid 
It's my first time to see a project leader is impolite and irresponsible manner in GitHub. 
I use a whole day around 12 hours to work on 3 languages (dialects) translations, and feedback is "modify the original English wordings" and "revert all things". Asking me either do all things or do nothing. 

Okay, I will close PR and NEVER contribute again any more on your project. 

**Shame on you!**